server/routes/index.coffee

```diff
@@ -2,11 +2,18 @@ mw = require '../middleware'
 
 module.exports.setup = (app) ->
   
+  app.all('/api/*', mw.api.clientAuth)
+  app.post('/api/users', mw.api.postUser)
+  app.get('/api/users/:handle', mw.api.getUser)
+  app.post('/api/users/:handle/o-auth-identities', mw.api.postUserOAuthIdentity)
+  app.post('/api/users/:handle/prepaids', mw.api.postUserPrepaid)
```

Comment:
In that case, could be `PUT`, and the parameter could be `ends`. That would match the User resource and change behavior. Can do.